tools/keyblob.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
```

Comment:
Since we are not using this at all in this PR, we may want this script in a separate ticket.